src/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html class="no-js" lang="">
```

Comment:
Is Modernizr or something that uses this class used in this project?

Reply:
Ha, I no. I just copied HTML5 Boilerplate.